tgbot/bot/conversations/main_application.py

```diff
@@ -0,0 +1,30 @@
+import logging
+import os
+
+from dotenv import load_dotenv
+from telegram import Update
+from telegram.ext import ApplicationBuilder, CommandHandler, ContextTypes
+
+load_dotenv()
+
+TELEGRAM_TOKEN = os.getenv('TELEGRAM_TOKEN')
+
+logging.basicConfig(
+    format='%(asctime)s - %(name)s - %(levelname)s - %(message)s',
+    level=logging.INFO
+)
+
+
+async def start(update: Update, context: ContextTypes.DEFAULT_TYPE):
+    await context.bot.send_message(
+        chat_id=update.effective_chat.id,
+        text='Тут текст старта')
+
+
+def main():
+    """Запуск бота."""
+    application = ApplicationBuilder().token(TELEGRAM_TOKEN).build()
+    start_handler = CommandHandler('start', start)
```

Comment:
тут не обязательно добавлять отдельную переменную 
```suggestion
    application.add_handler(CommandHandler('start', start)
```